### DISCUSSION TITLE
Using fbs to compile PyQt5 with oracledb throws an error in the executable file

### DISCUSSION THREAD

**Discussion by ghost** [oracle/python-oracledb#97] [Q&A] [answered]
I am trying to build a binary executable file out with Oracledb package for build a custom db application. When i run fbs run thats executing my application without any issue.

Even the fbs freeze also completing without any issue, when i execute my final binary executable its getting failed of missing module.
am not even getting any errors while running normal .py file, its all at binary file

FYI:
![image](https://user-images.githubusercontent.com/70656987/200392472-9c813bc0-9f68-4002-a6f6-8198aa15b1ef.png)

Please help me to get out of this

**Comment by anthony-tuininga** [comment, marked as answer]
Looks like you are using PyInstaller and you will need to specify that the `secrets` module needs to be included in the list of modules -- likely as a "hidden" import.

**Reply by ghost** [reply]
Hi @anthony-tuininga am using fbs which basically using Pyinstaller to build executables. I have check for the fbs documentation for importing modules. 
My doubt is that should i just import `secrets` or `oracledb.secrets`?

**Reply by anthony-tuininga** [reply]
You can just add the import to your script, even though you're not using it. It is a Python built-in module, not part of python-oracledb, so `secrets` is correct.

**Reply by anthony-tuininga** [reply]
The dependency section in https://build-system.fman.io/manual/ seems to be the place you want to go for help!

**Reply by ghost** [reply]
Thanks for sharing the document. Looks this gone fix my issue will have a try and let you know.

**Reply by ghost** [reply]
@anthony-tuininga Thanks for your support, that's resolved

**Comment by cjbj** [comment]
(For the record I deleted the identical question which was at https://github.com/oracle/python-oracledb/discussions/98)
